### PR TITLE
Updated "Dive Into Python" links.

### PR DESCRIPTION
**Pull request by TobeTek**
Revised Django documentation to highlight the official Python documentation as the primary resource for the language, replacing Dive into Python. Outdated links for Dive into Python were also updated

This PR builds on @The-Amoghavarsha 's work on #17353

### CODE REVIEW

File: docs/ref/django-admin.txt [hunk 1]
```diff
@@ -185,7 +185,7 @@ Example usage:
 
 .. django-admin-option:: --ignore PATTERN, -i PATTERN
 
-Ignores directories matching the given :mod:`glob`-style pattern. Use
+Ignores directories matching the given :py:mod:`glob`-style pattern. Use
```
Comment (TobeTek) [inline docs/ref/django-admin.txt:188]: Hi @felixxm !
Sorry for the ping, but in the previous PR #17353 , you mentioned [the `:py:` directive link should be used with links pointing to the official documentation](https://github.com/django/django/pull/17353#discussion_r1374118675).

I understand that to mean adding  `:py:` to be standard modules as I've done here. Would that be correct? 
Thanks!

Comment (felixxm) [reply docs/ref/django-admin.txt:188]: References that you've changed works even without `:py:`. Please revert these changes.

I was only talking about using `:py:` for ` _import search path`.

**Review by thibaudcolas — COMMENTED** [review]
Looking good! Preview site: https://17756--django-docs-preview.netlify.app/

Note there is also a link to https://www.diveinto.org/python3/ in the AUTHORS file, which probably should be updated as well?

**Comment by TobeTek** [comment]
> Note there is also a link to https://www.diveinto.org/python3/ in the AUTHORS file, which probably should be updated as well?

Updated!

**Comment by felixxm** [comment]
@TobeTek Thanks :+1: Welcome aboard :boat: 

I pushed tiny edits.

**Pull request merged**